### PR TITLE
KFSPTS-22886 create concur admin screen to get a new request and refresh token

### PR DESCRIPTION
**Pull request by jhulslander**
Please note there is a related non prod SQL PR

### CODE REVIEW

**Comment by ajd299** [comment]
Please see my comments. Also, what is the expected behavior if Concur is down?

**Review by ajd299 — CHANGES_REQUESTED** [review]

File: src/main/java/edu/cornell/kfs/concur/web/struts/ConcurManageRefreshTokenAction.java
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/concur/web/struts/ConcurManageRefreshTokenAction.java:97]: ok

File: src/main/java/edu/cornell/kfs/concur/web/struts/ConcurManageRefreshTokenAction.java
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/concur/web/struts/ConcurManageRefreshTokenAction.java:101]: ok

File: src/main/resources/edu/cornell/kfs/concur/cu-spring-concur.xml
Comment (jhulslander) [reply src/main/resources/edu/cornell/kfs/concur/cu-spring-concur.xml:26]: ok

File: src/main/resources/edu/cornell/kfs/concur/web/struts/form/datadictionary/ConcurManageRefreshTokenForm.xml
Comment (jhulslander) [reply src/main/resources/edu/cornell/kfs/concur/web/struts/form/datadictionary/ConcurManageRefreshTokenForm.xml:7]: ok

File: src/main/resources/edu/cornell/kfs/concur/web/struts/form/datadictionary/ConcurManageRefreshTokenForm.xml
Comment (jhulslander) [reply src/main/resources/edu/cornell/kfs/concur/web/struts/form/datadictionary/ConcurManageRefreshTokenForm.xml:37]: ok

File: src/main/webapp/jsp/concur/manageRefreshToken.jsp
Comment (jhulslander) [reply src/main/webapp/jsp/concur/manageRefreshToken.jsp:24]: no it shouldn't be there, I am surprised it didn't cause an error, I will remove

File: src/main/webapp/jsp/concur/manageRefreshToken.jsp
Comment (jhulslander) [reply src/main/webapp/jsp/concur/manageRefreshToken.jsp:14]: will remove

**Comment by jhulslander** [comment]
I've made a number of changes based on the feedback.  I've also adjusted the log level to debug on action class logging.  This is ready for review again

**Review by ajd299 — CHANGES_REQUESTED** [review]
I found one minor format issue. After that I think it is ready to be merged

**Comment by ajd299** [comment]
Looks good will merge when unit tests complete

**Comment by jhulslander** [comment]
I removed the square bracket, this is ready for review again

**Review by ajd299 — APPROVED** [review]

**Pull request merged**
